zio-k8s-client/src/main/scala/com/coralogix/zio/k8s/client/config/KeyManagers.scala

```diff
@@ -40,22 +38,11 @@ private object KeyManagers {
   ): ZIO[Any, Throwable, Array[KeyManager]] =
     for {
       keyStore <- getDefaultKeyStore
-      provider <- ZIO.attempt(new BouncyCastleProvider())
-
-      privateKey <- ZIO.scoped(loadKeyStream(key) flatMap { stream =>
-                      ZIO.attempt {
-                        val pemKeyPair = new PEMParser(new InputStreamReader(stream))
-                        val converter = new JcaPEMKeyConverter().setProvider(provider)
-                        pemKeyPair.readObject() match {
-                          case pair: PEMKeyPair   => converter.getPrivateKey(pair.getPrivateKeyInfo)
-                          case pk: PrivateKeyInfo => converter.getPrivateKey(pk)
-                          case other: Any         =>
-                            throw new IllegalStateException(
-                              s"Unexpected key pair type ${other.getClass.getSimpleName}"
-                            )
-                        }
-                      }
+
+      pemData    <- ZIO.scoped(loadKeyStream(key) flatMap { stream =>
+                      ZIO.attempt(scala.io.Source.fromInputStream(stream).mkString)
```

Comment:
isn't this leaking resources? usually when i'm using Source i use it with `ZIO.acquireRelease` to close it